addons/website_slides/i18n/website_slides.pot

```diff
@@ -3666,14 +3666,6 @@ msgstr ""
 msgid "Preview"
 msgstr ""
 
-#. module: website_slides
```

Comment:
Unless linked to a model field (aka when translations are unique), don't remove translations in pot files, just add the new ones. Idea is that translations should be available for people using the old version of the template (aka not doing -u).

Reply:
Hello @tde-banana-odoo ,
Point noted. Thanks for the review. 
I have applied the changes, can you please have a look ??

Reply:
actually do the same for existing sentences: add the new one, don't modify existing ones. This makes some noise but this allows to keep existing translations (same reason, people do not update necessarily views). In master this will be cleaned automatically when translations are generated before v17 freeze.

Reply:
Hello @tde-banana-odoo ,
Yes, indeed. I agree with you. Changes are done, can you please have a look ??